OWNERS_ALIASES

```diff
@@ -16,7 +16,6 @@ aliases:
   - yanweiguo
   channel-wg-leads:
   - matzew
-  - slinkydeveloper
```

Comment:
@slinkydeveloper Please revert the changes in this file and the other top-level owners_aliases again, the file is indeed auto-generated.